site/content/en/docs/handbook/mount.md

```diff
@@ -67,20 +67,26 @@ This directory may then be referenced from a Kubernetes manifest, for example:
 
 Some hypervisors, have built-in host folder sharing. Driver mounts are reliable with good performance, but the paths are not predictable across operating systems or hypervisors:
 
-| Driver | OS | HostFolder | VM |
-| --- | --- | --- | --- |
-| VirtualBox | Linux | /home | /hosthome |
-| VirtualBox | macOS | /Users | /Users |
-| VirtualBox | Windows | C://Users | /c/Users |
-| VMware Fusion | macOS | /Users | /mnt/hgfs/Users |
-| KVM | Linux | Unsupported | |
-| HyperKit | macOS | Supported |  |
-
-These mounts can be disabled by passing `--disable-driver-mounts` to `minikube start`.
-
-HyperKit mounts can use the following flags:
-`--nfs-share=[]`: Local folders to share with Guest via NFS mounts
-`--nfs-shares-root='/nfsshares'`: Where to root the NFS Shares, defaults to /nfsshares
```

Comment:
Clarify This Doc (and mention this is mostly depricate for Automaitc Mounts only happens with specific ways)

maybe move it out of this page

Reply:
I added vfkit and krunkit to the "Driver Mounts" section when they used --virtiofs-share, similar to hyperkit. Now that we use --mount they do not belong to this table.

Hyperkit is ok in this table since it is using different flags for mounting.

I think we need to add a section about "Virtiofs Mounts" documenting use of the --mount-string and listing the drivers supporting virtiofs.